packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -52,17 +52,13 @@ function PanZoomMesh() {
     [abscissaScale, ordinateScale, camera, visWidth, visHeight, invalidate]
   );
 
-  const onPointerDown = useCallback(
-    (evt: ThreeEvent<PointerEvent>) => {
-      const { sourceEvent, unprojectedPoint } = evt;
-      const { target, pointerId } = sourceEvent;
-      (target as Element).setPointerCapture(pointerId); // https://stackoverflow.com/q/28900077/758806
+  const onPointerDown = useCallback((evt: ThreeEvent<PointerEvent>) => {
+    const { sourceEvent, unprojectedPoint } = evt;
+    const { target, pointerId } = sourceEvent;
+    (target as Element).setPointerCapture(pointerId); // https://stackoverflow.com/q/28900077/758806
 
-      const projectedPoint = camera.worldToLocal(unprojectedPoint.clone());
-      startOffsetPosition.current = camera.position.clone().add(projectedPoint);
-    },
-    [camera]
-  );
+    startOffsetPosition.current = unprojectedPoint.clone();
```

Comment:
Rather than doing a mixture of projection/unprojection as before, I make `startOffsetPosition` the coordinates in the world space and work from here.